docs/resources/Channel.md

```diff
@@ -28,7 +28,7 @@ Represents a guild or DM channel within Discord.
 | last_pin_timestamp?                 | ?ISO8601 timestamp                                                          | when the last pinned message was pinned. This may be `null` in events such as `GUILD_CREATE` when a message is not pinned.                                                                    |
 | rtc_region?                         | ?string                                                                     | [voice region](#DOCS_RESOURCES_VOICE/voice-region-object) id for the voice channel, automatic when set to null                                                                                |
 | video_quality_mode?                 | integer                                                                     | the camera [video quality mode](#DOCS_RESOURCES_CHANNEL/channel-object-video-quality-modes) of the voice channel, 1 when not present                                                          |
-| message_count?                      | integer                                                                     | number of messages (not including the initial message or deleted messages) in a thread (if the thread was created before July 1, 2022, it stops counting at 50)                               |
+| message_count?                      | integer                                                                     | number of messages (not including the initial message or deleted messages) in a thread (only increments past 50 for messages sent after July 1, 2022)                                         |
```

Comment:
Not sure if it is necessary to explicitly mention the 50 limit; imo saying it's inaccurate is enough. Makes it easier to word and understand too.
```suggestion
| message_count?                      | integer                                                                     | number of messages (not including the initial message or deleted messages) in a thread. Threads created before July 1, 2022 may have an inaccurate message count.                                |
```